src/components/VersionSelector/VersionSelector.tsx

```diff
@@ -37,7 +37,11 @@ const VersionSelectorComponent = ({
     if (idx === selectedIdx) return setOpen(false);
 
     // navigate to resource version spec
-    const selectedResourceVersionUrl = `${rootUrl}/${resourceVersion}`;
+    let selectedResourceVersionUrl = `${rootUrl}/${resourceVersion}`;
+    const isPageTag = window.location.href.indexOf('#tag');
```

Comment:
NIT: the variable `isPageTag` makes it sound like the value will be a boolean, would it make sense to rename it to `pageTagIdx`?

Reply:
sounds good!